tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"moduleResolution": "node",
+		"noImplicitAny": true,
```

Comment:
NIT: That's already enabled by `"strict": true,`.
See: https://www.typescriptlang.org/docs/handbook/compiler-options.html
PS: I only know this b/c I was recently doing the skeleton project :)